model/sw_core.F90

```diff
@@ -1453,7 +1461,13 @@ subroutine d_sw(delpc, delp,  ptc,   pt, u,  v, w, uc,vc, &
 
      do j=js,je+1
         do i=is,ie+1
+!The following is for SA-3D-TKE
+          if(sa3dtke_dyco) then
+           damp2 = abs(dt)*dku3d_h(i,j) 
```

Comment:
Thanks also for remembering `abs(dt)` as this allows us to do our forward-backwards initialization.